### PR TITLE
Milestone 1: Initialize project with components

### PR DESCRIPTION
**Pull request by Debas-31**
# 👋 Micronaut Reviewer 

- [x] Initialize React app.
- [x] Add React-Redux (npm install react-redux).
- [x] The building blocks of your app should be set as re-usable components.
- [x] Add React Router and set two `<Route>`s and `<Link>`s for the app's navigation.
- [x] Styling is not required at this point.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const BooksPage = () => (
+  <>
+    <div className="books-container d-flex">
+      <h2>Book List</h2>
+      <button type="button">Remove</button>
```
Comment (Mihndim2020) [inline src/redux/books/books.js]: - [ ] As it is a requirement for this project(`see screenshot below`) and best practice, making sure building blocks of your app are re-usable components, please kindly add a `Book` component that; 
- Takes the book data as a prop and renders the title and the author
- Has a button for removing books(the button wouldn't remove the books *yet*, this is just a placeholder for now).
re-usable1
re-usable2

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    </div>
+    <input type="text" name="addBooks" placeholder="Book Title" />
+    <button type="button">Add Book</button>
```
Comment (Mihndim2020) [inline src/redux/books/books.js]: - [ ] As it is a requirement for this project(`see screenshot below`) and best practice, making sure building blocks of your app are re-usable components, please kindly add a `CreateNewBook` component that; 
- Renders a form with inputs for the title and the author of the book
- Has a button to submit( the button wouldn't add the book yet, this is just a placeholder for now )
re-usable1
re-usable2

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file,
:heavy_check_mark: Your project is running as expected.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @Debas-31, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**